web/src/main/java/edu/softserve/zoo/dto/LoginRequestDto.java

```diff
@@ -0,0 +1,39 @@
+package edu.softserve.zoo.dto;
+
+import edu.softserve.zoo.annotation.IrrespectiveDto;
+
+/**
+ * Represents information passed by a user during login attempt.
+ *
+ * @author Ilya Doroshenko
+ */
+@IrrespectiveDto
+public class LoginRequestDto {
```

Comment:
You've advised me to give up on SSL usage in project. But I suggest to count on SSL while designing security system. It's pure necessity!
If we won't use SSL then we'll have to implement all that encryption by our own.

And as with use of SSL all our messages will be encrypted, we may use plain text transmission. Of course back-end then hashes it and compares with hash stored in db.

So my point is:
1) if we want to transmit encrypted pass - it's best to use time-tested SSL\TLS
2) if we want to transmit client-side hashed pass - it's not giving us no security benefits